tests/test_add_news.py

```diff
@@ -2,11 +2,10 @@
 from pathlib import Path
 from types import SimpleNamespace
 
-from scikit_package.cli import add
 from scikit_package.cli.add import news_item
 
 
-def _setup_news_test_env(tmp_path):
+def _setup_news_test_env(tmp_path, mocker):
```

Comment:
overriding the hand-written value of `TEMPLATE_PATH`  from `news/TEMPLATE.rst` to `<tmp_path>/news/TEMPLATE.rst` using mocker. It